ephios/core/calendar.py

```diff
@@ -0,0 +1,36 @@
+from calendar import LocaleHTMLCalendar
+from datetime import date
+from itertools import groupby
+
+
+class ShiftCalendar(LocaleHTMLCalendar):
```

Comment:
Lets use `HTMLCalendar` as superclass and implement the formatting methods using django tools.